### PR TITLE
Chat spam

### PR DESCRIPTION
**Pull request by KernCS**
- Location spammer
- Did i ask?
- Clear Chat

### CODE REVIEW

**Comment by Khajiitos** [comment]
Nice dword

**Comment by SmokingDuckk** [comment]
Thank you so much! :D

**Comment by debooga** [comment]
interesting dword issue

**Comment by tirziz** [comment]
now i can afk and get low trust, thanks

File: Osiris/pstring.h [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+	pstring& operator+(const unsigned int i)
+	{
+		std::stringstream stream;
+		stream << *this;
+		stream << i;
+		*this = stream.str();
+		return *this;
+	}
+};
```
Comment (hyperxpro) [inline Osiris/pstring.h:32]: Add empty line here

**Comment by doggobot-owner** [comment]
Doesn't compile. Says something about 
static void to_json(json& j, const T& o, const T& dummy)
{
    if (o != dummy)
        j = o;
}
binary "!="

**Pull request closed without merging**